dashboard/test/models/reference_guide_test.rb

```diff
@@ -0,0 +1,32 @@
+require 'test_helper'
+
+class ReferenceGuideTest < ActiveSupport::TestCase
+  test "can create reference guide" do
+    reference_guide = create :reference_guide
+    assert reference_guide.id
+  end
+
+  test "reference guides are unique by slug in course version" do
+    create :reference_guide, key: 'hello/world', course_version_id: 1
+    create :reference_guide, key: 'hello/world', course_version_id: 2
+    assert_raises ActiveRecord::RecordInvalid do
+      create :reference_guide, key: 'hello/world', course_version_id: 1
+    end
+  end
+
+  test "reference guides seed from files correctly" do
+    co = create :course_offering, key: 'course'
+    cv = create :course_version, key: 'version', course_offering: co
+    guide = create :reference_guide, id: '1', display_name: 'World', key: 'hello/world', content: 'hello world', course_version_id: cv.id, position: 2
+    serialization = guide.serialize
+    previous_guide = guide.freeze
+    guide.destroy!
+
+    File.stubs(:read).returns(serialization.to_json)
+
+    new_guide_id = ReferenceGuide.seed_record("config/reference_guides/course_version/hello/world.json")
```

Comment:
would you consider changing "course_version" to something like "csp_2022", or "csx_1999", or something that gives a better idea of what this path part represents? "course_version" is a little confusing because the code which generates this suggests that course_version.key is only the second half of this token: https://github.com/code-dot-org/code-dot-org/blob/76312e7c8d17760a86fbf8c3bcb4b1af695ce359/dashboard/app/models/reference_guide.rb#L34-L35